src/uncrustify.cpp

```diff
@@ -38,10 +38,55 @@
 #include <vector>
 #include <deque>
 
+
+static const char *pcf_names[] =
```

Comment:
I don't like this move.
I prefer to have related definitions next to each other in the same file. Putting them in separate files increases the likelyhood of them getting out of sync.
I would be OK with an automated script that creates the pcf_names array based on the PCF macros.
That is done for other items.